### PR TITLE
feat: Add tokens for Page Indicator

### PR DESCRIPTION
**Pull request by magnuh**

### CODE REVIEW

File: tokens/blocket.se/pageindicator.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+token: maps
+
+color:
+  pageindicator:
```
Comment (AnnaRybkina) [inline tokens/blocket.se/pageindicator.yml:5]: Should this be one word? Based on the token `color-page-indicator-background` shouldn't it be 👇 
```suggestion
  page:
    indicator:
```

Comment (magnuh) [reply tokens/blocket.se/pageindicator.yml:5]: This has been discussed here: https://sch-chat.slack.com/archives/C050N3NQ204/p1682339440165969 🙂 
Dividing it like that would say that there is something named page which has a child called indicator (and potentially could have other children), which is not the case here... the component name is "page indicator"... so we need to decide how to handle names which has spaces in them. A better approach would perhaps have been to separate with underscores, like `color_page-indicator_background_hover`, and that way be able to use dashes for spaces in names.

Comment (AnnaRybkina) [reply tokens/blocket.se/pageindicator.yml:5]: Fyi, We have support for _ in the adiomatic script but it handles it the same as -. We can always change that so it makes sense for us

Comment (BalbinaK) [reply tokens/blocket.se/pageindicator.yml:5]: Ah, when I suggested joining longer component names I haven't thought about how that translates to @adidick's tokens structure. Maybe you could align with him and let us know the result of your findings 😊

Comment (magnuh) [reply tokens/blocket.se/pageindicator.yml:5]: Ok. 👍  ...that's what we in Sweden would call a #bajsmacka 😅 😆

**Review by BalbinaK — APPROVED** [review]

File: tokens/blocket.se/pageindicator.yml (hunk 1 above)
Comment (magnuh) [reply tokens/blocket.se/pageindicator.yml:5]: Decision was made to combine the words in multi word components name to a single word. ✔️ 😄

**Pull request merged**